plugins.json

```diff
@@ -215,7 +215,7 @@
         "state": "development",
         "plugin_info": "https://raw.githubusercontent.com/greschd/aiida-optimize/master/setup.json",
         "code_home": "https://github.com/greschd/aiida-optimize",
-        "documentation_url": "https://aiida-strain.readthedocs.io",
+        "documentation_url": "https://aiida-optimize.readthedocs.io",
         "pip_url": "aiida-strain"
```

Comment:
there's a few other places, perhaps just search for aiida-strain

Reply:
Oh yeah, and here I thought I'd checked it by eye...